homeassistant/core.py

```diff
@@ -141,15 +141,15 @@ def is_running(self) -> bool:
     def start(self) -> None:
         """Start home assistant."""
         # Register the async start
-        self.loop.create_task(self.async_start())
+        self.add_job(self.async_start())
 
         # Run forever and catch keyboard interrupt
         try:
             # Block until stopped
             _LOGGER.info("Starting Home Assistant core loop")
             self.loop.run_forever()
         except KeyboardInterrupt:
-            self.loop.create_task(self.async_stop())
+            self.add_job(self.async_stop())
```

Comment:
It actually works because we only start tracking tasks inside `self.async_stop`. But I'll revert it anyway, let core internal stuff use `loop.create_task`. I can see it going wrong in the future.